install.sh

```diff
@@ -5,7 +5,7 @@ set -uo pipefail
 
 main() {
   REQUIRED_BASH_VERSION="3.2.57"
-  REQUIRED_GIT_VERSION="2.18.0"
+  REQUIRED_GIT_VERSION="2.18.5"
```

Comment:
In werf source code I see the constraint as 2.18:
https://github.com/werf/werf/blob/0a509618bf8ba3734b7c942486e87895200ead6d/pkg/true_git/init.go#L15

Is there a reason for this change? If not, then revert, it might affect people who have Git 2.18.0-2.18.4 installed.